scripts/validate-commit-msg.js

```diff
@@ -16,7 +16,7 @@ var util = require('util');
 
```

Comment:
Is this file used any more (with `init-repo.sh` gone)? We could as well get rid of it altogether.